packages/argo-checkout/src/extension-points/api/post-purchase/post-purchase.ts

```diff
@@ -159,13 +169,19 @@ interface UpdatedLineItem {
   quantity: number;
 }
 
-interface UpdatedPurchase {
+interface AddedShippingLine {
+  priceSet: MoneyBag;
```

Comment:
Another docs issue where it lists `price` as part of this (guessing that's wrong)

Reply:
Yeah we should remove `price` from the doc, there is both `priceSet` and `price` there.

Reply:
Done. ✅